### PR TITLE
Improving SEO

### PR DESCRIPTION
**Pull request by poppyseedDev**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## DESCRIPTION
removed meta tags
shortened the description
added config.ts settings

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] contributes to  #95 
- [x] contributes to #100  
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [ ] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

**Comment by poppyseedDev** [comment]
@yangwao

File: developers/core-concepts/polkadot/parachains/kodadot-substrate-api.md [hunk 1]
```diff
@@ -3,10 +3,7 @@ title: KodaDot Substrate API Overview
 head:
   - - meta
     - name: description
-      content: The `sub-api` is a powerful utility developed by KodaDot for managing connections with multiple parachains in a network. This documentation provides an overview of its key features and shows how to use them effectively. Learn how to install the `sub-api` package, use the `ApiFactory` and `InstaApi` classes, leverage helper functions, hooks, and types, and see usage examples for fetching chain properties and listening for the `connect` event. The `sub-api` package is an integral part of KodaDot's developer toolbox and should simplify the task of connecting to and interacting with multiple parachains.
-  - - meta
-    - name: keywords
-      content: KodaDot, Substrate API, sub-api, utility, connections, parachains, network, overview, features, installation, ApiFactory, InstaApi, helper functions, hooks, types, usage examples, chain properties, connect event, developer toolbox, connecting, interacting
+      content: The `sub-api` is a powerful utility developed by KodaDot for managing connections with multiple parachains in a network.... Learn more.
```
Comment (yangwao) [inline developers/core-concepts/polkadot/parachains/kodadot-substrate-api.md:6]: remove ``

**Review by yangwao — APPROVED** [review]
